src/main/views/common/select-aml-supervisory-bodies/select-aml-supervisory-bodies.njk

```diff
@@ -7,6 +7,18 @@
         {% if acspType == "SOLE_TRADER" %}
            {% include "partials/user_name.njk" %}
         {% endif %}
+
+        {% set ITEMS = [] %}
+        {% for key,value in AMLSupervisoryBodies %}
+          {% if value is string %}
```

Comment:
Looks good, just checking why this check here?

Reply:
enum in typescript is behaving differently. it is like below.
{
0: aml1,
1: aml2,
3: aml3,
aml1: 0,
aml2: 1,
aml3: 3
}